### ISSUE TITLE
Invalid XML supplied by Icecast statistics

### ISSUE THREAD

**Issue by kenrestivo** [xiph/Icecast-Server#3]
Icecast version  2.3.3-2ubuntu1.14.04.1   , when reporting statistics via API calls to http://%s:%d/admin/stats.xml, returns this invalid XML:

```xml
<icestats><source mount="/radio"><Listeners>1</Listeners><listener><IP>23.2.9.2</IP><UserAgent>Mozilla/5.0 (iPhone; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Mobile/12F70 [FBAN/FBIOS;FBAV/46.0.0.54.156;FBBV/18972819;FBDV/iPhone5,3;FBMD/iPhone;FBSN/iPhone OS;FBSV/8.3;FBSS/2; FBCR/AT&T;FBID/phone;FBLC/en_US;FBOP/5]</UserAgent><Connected>0</Connected><ID>2634887</ID></listener></source></icestats>
```

Note the unescaped &T; which is an undefined XML entity "T", and causes parsers to explode.

Expected behavior: text inside blocks should be html escaped or enclosed in a CDATA block, if I remember my XML correctly.

**Comment by dm8tbr** [comment]
Please verify on latest Icecast. A lot has changed since 2.3.3.
https://wiki.xiph.org/Icecast_Server/Installing_latest_version_%28official_Xiph_repositories%29

**Comment by dm8tbr** [comment]
Had some time to spare. Verified on latest trunk.
Moving ticket to [trac.xiph.org](trac.xiph.org), where we track all Icecast issues:
https://trac.xiph.org/ticket/2255#ticket

**Comment by ePirat** [comment]
This is fixed now, see commit d739c65. Thanks for the report!

**Comment by kenrestivo** [comment]
Great, thanks for the quick fix!
